modules/apps/object/object-info-api/src/main/java/com/liferay/object/info/item/provider/util/ObjectEntryInfoItemValuesProviderUtil.java

```diff
@@ -91,17 +91,17 @@ public static List<InfoFieldValue<Object>> getInfoFieldValues(
 				continue;
 			}
 
-			_addInfoFieldValue(
-				dlAppLocalService, dlURLHelper, infoFieldValues,
-				listTypeEntryLocalService, objectEntryLocalService, objectField,
-				objectFieldInfoFieldConverter,
-				ObjectField.class.getSimpleName(),
-				objectRelationshipLocalService, themeDisplay,
-				values.get(objectField.getName()));
-
 			if (!objectField.compareBusinessType(
 					ObjectFieldConstants.BUSINESS_TYPE_RELATIONSHIP)) {
 
+				_addInfoFieldValue(
+					dlAppLocalService, dlURLHelper, infoFieldValues,
+					listTypeEntryLocalService, objectEntryLocalService,
+					objectField, objectFieldInfoFieldConverter,
+					ObjectField.class.getSimpleName(),
+					objectRelationshipLocalService, themeDisplay,
+					values.get(objectField.getName()));
+
 				continue;
```

Comment:
@vngoliveira we should include the info field value for relationship fields. Please try to fix the code where the NPE is happening. 

In the [SiteObjectScopeProviderImpl](https://github.com/liferay/liferay-portal/blob/bdae4717b6a049554992063ab13a1fb1035d1bac/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/scope/SiteObjectScopeProviderImpl.java#L35), you can try to do something like:

```
@Override
public long getGroupId(HttpServletRequest httpServletRequest)
	throws PortalException {

	ThemeDisplay themeDisplay =
		(ThemeDisplay)httpServletRequest.getAttribute(
			WebKeys.THEME_DISPLAY);

	if (themeDisplay != null) {
		return themeDisplay.getScopeGroupId();
	}

	ServiceContext serviceContext =
		ServiceContextThreadLocal.getServiceContext();

	if (serviceContext == null) {
		return 0;
	}

	return serviceContext.getScopeGroupId();
}
```